src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationSource.cs

```diff
@@ -39,9 +39,9 @@ public abstract class FileConfigurationSource : IConfigurationSource
         public int ReloadDelay { get; set; } = 250;
 
         /// <summary>
-        /// Will be called if an uncaught exception occurs in FileConfigurationProvider.Load.
+        /// Will be called if an uncaught exception occurs in Load.
         /// </summary>
-        public Action<FileLoadExceptionContext> OnLoadException { get; set; }
+        public Action<LoadExceptionContext> OnLoadException { get; set; }
```

Comment:
It would probably imply making this more of first class concept, even on IConfiguraitonProvider.